community_supplied/Interface/check-if-mtu-netconf.rule

```diff
@@ -0,0 +1,156 @@
+/*
+* Detects interface MTU values and notifies when anomalies
+* are found.
+*
+* Two input control detection
+*
+* 1) interface-name, is a regular expression that matches the
+* interfaces that you would like to monitor. By default it's
+* '.*' which matches all interfaces. Use something like 'ge.*'
+* to match only gigabit ethernet interfaces.
+*
+* 2)mtu-variable, is MTU recommended value that causes the rule
+* to report an anomaly. By default it is 1514. This rule will set
+* the dashboard color to green when interface MTU value is equal
+* to the recommended MTU value. Otherwise it is set to red.
+*/
+healthbot {
+    topic interface.statistics {
+        description "Monitors and notify interface statistics i.e. link state, flaps, neighbor state, input & output traffic and errors";
+        synopsis "Interface statistics analyzer";
+        rule check-if-mtu-netconf {
+            keys interface-name;
+            synopsis "Interface MTU analyzer";
+            description "Collects the interface MTU values periodically and notifies in case of anomalies";
+            /*
+            * Monitors the interface MTU values and notifies via the
+            * dashboard colors when the interface MTU value is equal
+            * to recommended MTU value, it is set to green. Otherwise
+            * it is set to red.
+            *
+            * Use interface-name as key for rule
+            *
+            *
+            * Sensor Configuration to get data from network devices.
+            *
+            */
+            sensor interface {
+                synopsis "Interfaces netconf based iAgent sensor definition";
+                description "iAgent sensor defined in yml file using netconf rpc/cli command and defined fields using pyez tables & views to collect data from network device";
+                iAgent {
+                    file IntStatsTable.yml;
+                    table IntStatsTable;
+                    frequency 60s;
+                }
+            }
+            /*
+            * Fields defined using sensor path. Map the longer sensor names
+            * to the shorter field names used in the rules.
+            */
+            field admin-status {
```

Comment:
"admin-status" field not being used. You can filter admin down interfaces using where condition in field "interface-name"